SafeFoodie_Fresh/lib/screens/authenticate/RestPassword.dart

```diff
@@ -0,0 +1,101 @@
+
+  @override
+  State<RestPassword> createState() => _RestPasswordState();
+}
+
+class _RestPasswordState extends State<RestPassword> {
+  late String email;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: const Color(0xff251f34),
+      resizeToAvoidBottomInset: true,
+      appBar: AppBar(
+        elevation: 0,
+        backgroundColor: const Color(0xff251f34),
+      ),
+      // Removed Unnecessary Container
+     body: Column(
+            mainAxisAlignment: MainAxisAlignment.start,
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              const Padding(
+                padding: EdgeInsets.fromLTRB(20, 20, 20, 8),
+                child: Text(
+                  'Reset Password',
+                  style: TextStyle(
+                      color: Colors.white,
+                      fontWeight: FontWeight.w600,
+                      fontSize: 25),
+                ),
+              ),
+              // added const
+             const SizedBox(
+                height: 30,
+              ),
+              // addedconst 
+             const Padding(
+                padding: const EdgeInsets.symmetric(horizontal: 20),
+                child: Text(
+                  'Email',
+                  style: TextStyle(
+                      color: Colors.white,
+                      fontWeight: FontWeight.w600,
+                      fontSize: 13),
+                ),
+              ),
+              // added const 
+             const SizedBox(
+                height: 10,
+              ),
+              Padding(
+                padding: const EdgeInsets.symmetric(horizontal: 20),
+                child: TextField(
+                  style: const TextStyle(
+                    color: Colors.white,
+                    fontWeight: FontWeight.w400,
+                  ),
+                  keyboardType: TextInputType.emailAddress,
+                  cursorColor: Colors.white,
+                  //added const 
+                  decoration: const InputDecoration(
+                      border: InputBorder.none,
+                      fillColor: const Color(0xff3B324E),
+                      filled: true,
+                      focusedBorder: const OutlineInputBorder(
+                        borderSide:
+                            BorderSide(color: Color(0xff14dae2), width: 2.0),
+                        borderRadius: BorderRadius.all(Radius.circular(20.0)),
+                      )),
+                  onChanged: (value) {
+                    setState(() {
+                      email = value;
+                    });
+                  },
+                ),
+              ),
+              // added const
+              const SizedBox(
+                height: 10,
+              ),
+              Center(
+                  child: ElevatedButton(
+                      onPressed: () async {
+                        await FirebaseAuth.instance
```

Comment:
On pressed we make a call to firebase where they take our email and able to send a rest password email to users and then after that it will return back to the main page